components/cam/bld/config_files/definition.xml

```diff
@@ -340,6 +340,9 @@ Switch to enable or disable fractional cloudiness in m2005 microphysics in CRM:
 <entry id="MMF_microphysics_scheme" valid_values="sam1mom,m2005" value="sam1mom">
 SPCAM Cloud microphysics type.
 </entry>
+<entry id="crm" valid_values="sam,samxx" value="sam">
+SPCAM model
```

Comment:
gah! I thought I removed all mentions of SPCAM! Anyway, just want to register my disgust. No need to change this.